assets/aws/files/install-hardened.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+if [[ "${DEBUG:-false}" == "true" ]]; then
+    set -x
```

Comment:
The script is invoked with `-x` from the hcl config. Is this doing anything more? I guess it's for testing manually? I usually just run `bash -x script` when I want to see all the commands run for debug purposes.